pkg/graph/model/uuid.go

```diff
@@ -0,0 +1,26 @@
+package model
+
+import (
+	"errors"
+	"io"
+
+	"github.com/99designs/gqlgen/graphql"
+	"github.com/google/uuid"
+)
+
+// MarshalUUID allows uuid to be marshalled by graphql
+func MarshalUUID(id uuid.UUID) graphql.Marshaler {
+	return graphql.WriterFunc(func(w io.Writer) {
+		_, _ = io.WriteString(w, id.String())
```

Comment:
I think this needs to quote the string to avoid an issue with JSON serialization:

```suggestion
		_, _ = io.WriteString(w, strconv.Quote(id.String()))
```